types/auth0/index.d.ts

```diff
@@ -8,6 +8,8 @@
 //                 Anthony Messerschmidt <https://github.com/CatGuardian>
 //                 Johannes Schneider <https://github.com/neshanjo>
 //                 Meng Bernie Sung <https://github.com/MengRS>
+//                 Léo Haddad Carneiro <https://github.com/Scoup>
```

Comment:
This will require sign-off from Scoup as not coming from PR author or commmit author

/cc @Scoup

Reply:
Hi, we worked together on this feature but choose to only one open the PR.